conda/gateways/repodata/__init__.py

```diff
@@ -581,7 +581,7 @@ def load_state(self):
             self.load(state_only=True)
         except (FileNotFoundError, json.JSONDecodeError) as e:
             if isinstance(e, json.JSONDecodeError):
-                warnings.warn(f"{e.__class__.__name__} loading {self.cache_path_state}")
+                log.warning(f"{e.__class__.__name__} loading {self.cache_path_state}")
```

Comment:
The main difference is that `warnings.warn` will only show up once for the same parameter, while `log.warning` will emit to the configured log output every occurrence. More infos: https://stackoverflow.com/a/37979724